.github/workflows/build_source_release.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Source Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    defaults:
+      run:
+        shell: bash
```

Comment:
I don't think this block is needed since bash is the default on linux.

Reply:
ok, will amend after a test.